src/js/apps/patients/patient/sidebar/sidebar_app.js

```diff
@@ -8,110 +8,12 @@ import { SidebarView } from 'js/views/patients/patient/sidebar/sidebar_views';
 export default App.extend({
   onBeforeStart({ patient }) {
     const currentOrg = Radio.request('bootstrap', 'currentOrg');
-
-    this.orgEngagementEnabled = currentOrg.getSetting('engagement');
```

Comment:
I may be missing somewhere, but it looks like this setting change is no longer checked anywhere? I know you handle errors from `GET /patients/{patientId}/engagement-status` gracefully in the widget, but we should still check if the setting is enabled before even trying to hit that endpoint. 

The reason being is that there is an edge case where we accidentally add the engagement widget to the sidebar, with it disabled. This could, very briefly, show the engagement widget then disappear. This would not only look odd, but Will has said indicated in the past that we don't want customers who don't have it to even know it's a thing unless we tell them about it. 

If I'm missing it somewhere, sorry for the long comment and carry on.

Reply:
We used that flag to filter out the engagement widget from the static set, not to prevent rendering an existing widget.  That'd be a bit unnecessarily complicated for an edge case I think.  If we're worried about the edge case of adding it accidentally, maybe the endpoint , as the source of truth, should filter it out before sending it for display.